types/nightwatch/test/testElementCommands.ts

```diff
@@ -0,0 +1,505 @@
+            .clearValue('input[type=text]');
+        isNull(result);
+    });
+});
+
+//
+// .click
+//
+describe('click Command demo', function() {
+    test('demo test', function() {
+        browser
+            .url('https://google.com')
+            .waitForElementVisible('input[type=text]')
+            .setValue('input[type=text]', 'nightwatch.js')
+            .click('input[type=submit]', function(result) {
+                isNightwatchAPI(this);
+            });
+    });
+
+    test('async demo test', async function(browser) {
+        const result = await browser
+            .url('https://google.com')
+            .waitForElementVisible('input[type=text]')
+            .setValue('input[type=text]', 'nightwatch.js')
+            .click('input[type=submit]');
+        isNull(result);
+    });
+});
+
+//
+// .getAttribute
+//
+describe('getAttribute command demo', function() {
+    before(browser => browser.url('https://www.google.com/'));
+
+    test('demo test', function() {
+        browser
+            .getAttribute('input[type=text]', 'title' , function(result) {
+                isNightwatchAPI(this);
+            });
+    });
+
+    test('async demo test', async function(browser) {
+        const result = await browser.getAttribute('input[type=text]', 'title');
+        isStringOrNull(result);
+    });
+
+    after(browser => browser.end());
+});
+
+//
+// .getCssProperty
+//
+describe('getCssProperty command demo', function() {
+    before(browser => browser.url('https://www.google.com/'));
+
+    test('demo test', function() {
+        browser
+            .getCssProperty('input[type=text]', 'background-color' , function(result) {
+                isNightwatchAPI(this);
+            });
+    });
+
+    test('async demo test', async function(browser) {
+        const result = await browser.getCssProperty('input[type=text]', 'background-color');
+        isString(result);
+    });
+
+    after(browser => browser.end());
+});
+
+//
+// .getElementSize
+//
+describe('getElementSize command demo', function() {
+    test('demo test', function() {
+        browser
+            .url('https://www.ecosia.org/')
+            .getElementSize('#navbartop', function(result) {
+                isNightwatchAPI(this);
+            });
+    });
+
+    test('async demo test', async function(browser) {
+        const result = await browser.url('https://www.ecosia.org/').getElementSize('#__nuxt');
+        isElementSize(result);
+    });
+});
+
+//
+// .getLocation
+//
+describe('getLocation command demo', function() {
+    test('demo test', function() {
+        browser
+            .url('https://www.ecosia.org/')
+            .getLocation('#__nuxt', function(result) {
+                isNightwatchAPI(this);
+            });
+    });
+
+    test('async demo test', async function(browser) {
+        const result = await browser.url('https://www.ecosia.org/').getLocation('#__nuxt');
+        console.log(result);
```

Comment:
Please replace all `console.log` in this file with actual type checks:
```suggestion
        isNightwatchSizeAndPosition(result);
```